manifests/init.pp

```diff
@@ -221,7 +221,7 @@
 #
 # $dns_reverse::                DNS reverse zone name
 #
-# $dns_server::                 Address of DNS server to manage
+# $dns_server::                 Address of DNS server to manage, set to empty string to send updates to master server
```

Comment:
I think master server is ambiguous and would prefer `set to an empty string to let nsupdate automatically determine the master server`.

Reply:
Nice thanks I am bad in this.